CMakeLists.txt

```diff
@@ -1,4 +1,4 @@
-cmake_minimum_required(VERSION 3.21)
+cmake_minimum_required(VERSION 3.26)
```

Comment:
Doesn't appear this was quite enough to get the info to display in the logs. I'm wondering if the version needs bumped in both wxdata/CMakeLists.txt and wxdata/wxdata.cmake as well.

If this still doesn't do it I'm willing to let it go.